### PR TITLE
Shuffle the dataset while fitting by default

### PR DESCRIPTION
**Pull request by lilleswing**
Still evaluate dataset in order by default on prediction.

### CODE REVIEW

File: deepchem/models/tensorgraph/tensor_graph.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def fit(self,
           max_checkpoints_to_keep=5,
           checkpoint_interval=1000):
     return self.fit_generator(
-        self.default_generator(dataset, epochs=nb_epoch),
+        self.default_generator(dataset, epochs=nb_epoch, deterministic=False),
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/tensor_graph.py]: @miaecle this might change some of the benchmark results though......

**Comment by rbharath** [comment]
In general, I'm in favor of shuffling by default.

@lilleswing Could you do a sanity check Tox21 run to make sure numbers look decent?

@miaecle, @lilleswing: I think maintaining the benchmark table on GitHub is going to become onerous. I propose we shift it to deepchem.io. We can set up a (weekly? biweekly?) Jenkins job that runs the full benchmarks and automatically pushes them to AWS

**Comment by lilleswing** [comment]
@rbharath where is the code for the deepchem.io website?  

We could do something like....

1) Run full benchmark -> store json in s3 public bucket
2) pull the s3 public bucket with js and display html table

**Comment by lilleswing** [comment]
@rbharath the tox21 are still within threshold after shuffling.

**Comment by rbharath** [comment]
@lilleswing All code for deepchem.io is in this repo :). doc/ holds all the Python code for generating sphinx docs, and https://github.com/deepchem/deepchem/blob/master/devtools/travis-ci/push-docs-to-s3.py pushes built docs to S3 after merges

File: deepchem/models/tensorgraph/tensor_graph.py (hunk 1 above)
Comment (rbharath) [inline deepchem/models/tensorgraph/tensor_graph.py]: Might be good to make deterministic a kwarg set to True. That way, users can turn off if desired

**Comment by rbharath** [comment]
LGTM

**Pull request merged**